trampoline/src/main/java/org/ernest/applications/trampoline/buildtool/MavenRunner.java

```diff
@@ -0,0 +1,34 @@
+package org.ernest.applications.trampoline.buildtool;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class MavenRunner extends BuildToolCommandRunner {
+
+    public final static String MAVEN_HOME = "M2_HOME";
+
+    private String mavenPath;
+
+    public String getMavenPath() {
+        return mavenPath;
+    }
+
+    public void setMavenPath(String mavenPath) {
+        this.mavenPath = mavenPath;
+    }
+
+    @Override
+    protected List<String> getCommand(String port, String[] parsedArguments) {
```

Comment:
Doesn't work on windows:

2018-03-05 20:37:52.320 ERROR 9804 --- [nio-8080-exec-4] o.e.a.trampoline.services.FileManager    : Failed to run Gradle build tool for microservice [id: af760d56-f0bc-44c4-95a2-f135cf1525ee name: ms_maven]
2018-03-05 20:37:52.381 ERROR 9804 --- [nio-8080-exec-4] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is org.ernest.applications.trampoline.exceptions.RunningMicroserviceScriptException: Cannot run program "C:\workarea\tools\apache-maven-3.3.9\bin\mvn" (in directory "C:\Users\Ernest\Documents\GitHub\Trampoline\microservice-example-maven"): CreateProcess error=193, %1 no es una aplicaci├│n Win32 v├ílida] with root cause

org.ernest.applications.trampoline.exceptions.RunningMicroserviceScriptException: Cannot run program "C:\workarea\tools\apache-maven-3.3.9\bin\mvn" (in directory "C:\Users\Ernest\Documents\GitHub\Trampoline\microservice-example-maven"):

Reply:
Hmmm . . . I will have to check it out.